### PR TITLE
doc: sig-java added

### PR DESCRIPTION
**Pull request by wey-gu**

### CODE REVIEW

**Comment by wey-gu** [comment]
cc: @Anyzm @CorvusYe @Young-Flash

**Comment by Anyzm** [comment]
> cc: @Anyzm @CorvusYe @Young-Flash

I have no questions about this file， what can i do for you？

**Comment by wey-gu** [comment]
> > cc: @Anyzm @CorvusYe @Young-Flash
> 
> I have no questions about this file， what can i do for you？

Could you please help review it and also share me your email so that I could add you into the mail list for the sig?

**Comment by CorvusYe** [comment]
CorvusY@foxmail.com

**Review by Nicole00 — APPROVED** [review]

File: sig/java/README.md [hunk 1]
```diff
@@ -0,0 +1,83 @@
+# Special Interest Group Java
+
+We ❤️ Java and NebulaGraph.
+
+## Meetings
+
```
Comment (wey-gu) [inline sig/java/README.md:6]: comment from @tian-zhong , Scope to be added

File: sig/java/README.md [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+* [@Nicole00](https://github.com/Nicole00), vesoft
+
+### Secretary
+
+Lobby boy of the sig.
+
+* [@wey-gu](https://github.com/wey-gu), vesoft
+
+### Technical Leads
```
Comment (wey-gu) [inline sig/java/README.md:25]: suggested by @tian-zhong to remove Tech Leads here.

File: sig/java/README.md [hunk 3]
```diff
@@ -0,0 +1,83 @@
...
+
+## Subprojects
+
+The following *subprojects* are owned by sig-java:
+
+### Graph-Ocean
+
+NebulaGraph Java ORM
+
+- **Owners:**
```
Comment (wey-gu) [inline sig/java/README.md:49]: replace owner with PTL.

File: sig/java/README.md [hunk 4]
```diff
@@ -0,0 +1,83 @@
...
+
+## Contact
+
+- [Mailing list](nebulagraph-sig-java@googlegroups.com)
+  - https://groups.google.com/g/nebulagraph-sig-java/
+  - nebulagraph-sig-java@googlegroups.com
+
+- [Open Community Issues/PRs](https://github.com/vesoft-inc/nebula-community/issues)
+
+## Subprojects
```
Comment (wey-gu) [inline sig/java/README.md:41]: Maintainer list to be added

File: sig/java/README.md [hunk 5]
```diff
@@ -0,0 +1,83 @@
...
+- **Repo**:
+  - https://github.com/nebula-contrib/ngbatis
+
+### Nebula-Java
+
+- **Owners**:
+  - [@Nicole00](https://github.com/Nicole00)
+- **Repo**:
+  - https://github.com/vesoft-inc/nebula-java
+
```
Comment (wey-gu) [inline sig/java/README.md:79]: @Nicole00 suggested adding Nebula Java in the actions repo

**Review by QingZ11 — APPROVED** [review]

**Comment by Anyzm** [comment]
感谢您的来件，邮件已收到。

**Comment by Anyzm** [comment]
感谢您的来件，邮件已收到。

**Pull request merged**